react/src/components/buttons/SprkButton.js

```diff
@@ -141,6 +143,10 @@ SprkButton.propTypes = {
    * The actual value is what is applied to the href attribute.
    */
   href: PropTypes.string,
+  /**
+   * A ref passed in will be attached to the button component.
```

Comment:
Verify if this is in the release notes @guern1kn

Reply:
If it's not yet, we'll just want to add a bit about how it's a new feature.

Reply:
Added.